lib/ansible/modules/cloud/amazon/ec2_tag.py

```diff
@@ -33,8 +34,17 @@
     description:
       - a hash/dictionary of tags to add to the resource; '{"key":"value"}' and '{"key":"value","key":"value"}'
     required: true
-
-author: "Lester Wade (@lwade)"
+  purge_tags:
+    description:
+      - Whether unspecified tags should be removed from the resource.
+      - "Note that when combined with C(state: absent), specified tags with non-matching values are not purged."
```

Comment:
I think this behavior should be shown in the examples since it is tricky to explain.

Reply:
Do we need an additional example beyond lines 80-87? (Or is there anything about it you would change?)
```
+- name: Remove all tags except for Name from an instance
+  ec2_tag:
+    region: eu-west-1
+    resource: i-xxxxxxxxxxxxxxxxx
+    tags:
+        Name: ''
+    state: absent
+    purge_tags: true
```